x/ecocredit/server/core/new_credit_type.go

```diff
@@ -0,0 +1,31 @@
+package core
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	api "github.com/regen-network/regen-ledger/api/regen/ecocredit/v1"
+	"github.com/regen-network/regen-ledger/x/ecocredit/core"
+)
+
+// NewCreditType adds a new credit type to the network
+func (k Keeper) NewCreditType(ctx sdk.Context, ctp *core.CreditTypeProposal) error {
```

Comment:
I think this should be `AddCreditType` rather than `NewCreditType`.

Reply:
updated https://github.com/regen-network/regen-ledger/pull/1015/commits/975f386c657b1f2d09e3fd1b561f57727a6a1d10